quinn/src/endpoint.rs

```diff
@@ -318,6 +338,19 @@ impl Endpoint {
     }
 }
 
+// Struct for holding the stats returned from Endpoint::stats
```

Comment:
This should be an actual docstring (`///`) without the trailing empty lines.

There should also be a docstring for each of the public fields, in particular taking care to document which of these are monotonically increasing vs other behavior (perhaps we can reflect the difference in the field names, too).

Reply:
"Struct for holding the stats returned from Endpoint::stats" is a very mechanical description. Maybe something more like "Statistics on [`Endpoint`] activity"?

Also, note that I requested you document each field here.